packages/jellyfish-transaction/src/script/dftx/dftx_masternode.ts

```diff
@@ -7,9 +7,8 @@ import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
  * CreateMasterNode DeFi Transaction
  */
 export interface CreateMasterNode {
-  type: number // ----------------------------------| 1 byte, if 0x01 p2pkh, otherwise = p2wpkh
-  collateralPubKeyHash: string // ------------------| VarUInt{20 bytes}
-  operatorPubKeyHash?: string // -------------------| VarUInt{20 bytes}, p2pkh only, optional, use collateral pkh as default
```

Comment:
https://github.com/DeFiCh/ain/blob/85360ad432ae8c5ecbfbfc7d63dd5bc6fe41e875/src/masternodes/mn_checks.h#L157-L159
only one address for operating